src/units/gender.cpp

```diff
@@ -17,26 +17,79 @@
 
 #include <ostream>
 
+namespace {
+	/// Standard string id (not translatable) for FEMALE
+	const std::string s_female("female");
+	/// Standard string id (not translatable) for MALE
+	const std::string s_male("male");
```

Comment:
I'm pretty sure these need to be marked translatable. Obviously you can't use the `_()` syntax, but there's an alternate syntax that marks it as translatable for gettext without implying an actual lookup (you'd then look it up later, probably via the `t_string` constructor).

Reply:
These were always untranslatable, see current `src/unit/race.cpp`. They are simply identifiers, used for WML keys and such. The actual translatable versions are in different locations. Refactoring that might be a good idea, but is out of scope of this PR.

Reply:
Ah, okay.